### DISCUSSION TITLE
TypeError

### DISCUSSION THREAD

**Discussion by IceDragons37** [theOehrly/Fast-F1#185] [Q&A]
The following line of code always gets flagged:
laps['LapTimeSeconds'] = laps['LapTime'].dt.total_seconds()

And the corresponding error is "TypeError: 'NoneType' object is not subscriptable". Could you help me find a fix for the error? Thx

**Comment by theOehrly** [comment]
You need to provide more code than this if you want us to help you. That line itself is fine and should not cause an error. The problem is somewhere else.

In general, you should always provide a short example that can be run as is to reproduce the error.

**Reply by IceDragons37** [reply]
Is the following enough code or is more code needed to try and reproduce the error?

plotting.setup_mpl()
ff1.Cache.enable_cache('cache')
pd.options.mode.chained_assignment = None

race = ff1.get_session(2021, 'France', 'R')
laps = race.load()

laps['LapTimeSeconds'] = laps['LapTime'].dt.total_seconds()
laps = laps.loc[(laps['PitOutTime'].isnull() & laps['PitInTime'].isnull())]

q75, q25 = laps['LapTimeSeconds'].quantile(0.75), laps['LapTimeSeconds'].quantile(0.25)

**Reply by theOehrly** [reply]
Yes, it is, even though this example is still incomplete. You should include imports as well so one can simply copy and paste your example to test it.

Your problem is `laps = race.load()`. The method `.load` does not have a return value. Therefore, `laps` is None.

You can do `laps = race.laps` after calling `race.load()`

**Reply by IceDragons37** [reply]
It fixed the issue. Thank you so much for helping me
